data/brands/shop/chemist.json

```diff
@@ -10,6 +10,18 @@
     }
   },
   "items": [
+     {
+      "displayName": "Caddy's",
+      "locationSet": {"include": ["it"]},
+      "matchTags": ["shop/conveneience", "shop/houseware", "shop/cosmetics"],
+      "tags": {
+        "brand": "Caddy's",
+        "brand:wikidata": "Q108604630",
+        "name": "Caddy's",
+        "old_name": "Cad",
```

Comment:
Is this just to tag for a rebranding?

Reply:
yah looks like a 2017 rebrand: https://ovostudio.it/portfolio/cad-caddys
I'll move this to a matchName, since that was a while ago.